Submitted by:            Douglas R. Reno <renodr2002 at gmail dot com>
Date:                    2016-05-05
Initial Package Version: 2.40
Upstream Status:         Fixed
Origin:                  Fedora
Description:             Fixes building with GCC >= 6.1.0

diff -Naurp seamonkey-2.40.orig/mozilla/nsprpub/config/make-system-wrappers.pl seamonkey-2.40/mozilla/nsprpub/config/make-system-wrappers.pl
--- seamonkey-2.40.orig/mozilla/nsprpub/config/make-system-wrappers.pl	2016-01-18 20:47:56.000000000 -0600
+++ seamonkey-2.40/mozilla/nsprpub/config/make-system-wrappers.pl	2016-05-05 19:14:10.187489674 -0500
@@ -19,7 +19,9 @@ while (<STDIN>) {
     open OUT, ">$output_dir/$_";
     print OUT "#pragma GCC system_header\n";  # suppress include_next warning
     print OUT "#pragma GCC visibility push(default)\n";
+    print OUT "#define _GLIBCXX_INCLUDE_NEXT_C_HEADERS\n";
     print OUT "#include_next \<$_\>\n";
+    print OUT "#undef _GLIBCXX_INCLUDE_NEXT_C_HEADERS\n";
     print OUT "#pragma GCC visibility pop\n";
     close OUT;
 }
